cypress/integration/pages/articles/tests.js

```diff
@@ -29,40 +28,16 @@ const serviceHasCorrectlyRenderedParagraphs = service => service !== 'sinhala';
 
 const serviceHasTimestamp = service => ['news', 'urdu'].includes(service);
 
-// For testing important features that differ between services, e.g. Timestamps.
-// We recommend using inline conditional logic to limit tests to services which differ.
-export const testsThatAlwaysRun = ({ service, pageType, variant }) => {
+export const testsThatAlwaysRun = ({ service, pageType }) => {
```

Comment:
@jamesdonoh the `variant` argument was being used in the logic removed, so it made sense to remove it as the linter flagged it as it was an unused variable. The comment referred to mostly logic that differed across services but since we changed it to just check for the presence of timestamps across services and not how they differ, then that comment could become misleading somewhat

Reply:
I think it's more important to preserve consistency across all test files (compare frontPage and other page types to see how this structure is replicated) to make maintenance easier.

Reply:
@jamesdonoh we could leave the `testsThatAlwaysRun` block with the comment as it is relevant for test in that block and move e2e describe block to `testsThatFollowSmokeTestConfig`, what do you think ??

Reply:
Could do... please can you explain your reasoning for suggesting moving it to `testsThatFollowSmokeTestConfig` to make sure I understand?

Reply:
tests in `testsThatFollowSmokeTestConfig` are for testing `"features that differ across services but share a common logic"` so I figured since this logic won't change across services, we could add it there as well.